sdk/src/wallet/bindings/java/examples/src/CheckBalance.java

```diff
@@ -11,16 +11,19 @@
 
 public class CheckBalance {
     public static void main(String[] args) throws WalletException, InterruptedException, InitializeWalletException {
-        // This example assumes that a wallet has already been created using the ´SetupWallet.java´ example.
-        // If you haven't run the ´SetupWallet.java´ example yet, you must run it first to be able to load the wallet as shown below:
+        // This example assumes that a wallet has already been created using the
+        // ´SetupWallet.java´ example.
+        // If you haven't run the ´SetupWallet.java´ example yet, you must run it first
+        // to be able to load the wallet as shown below:
         Wallet wallet = new Wallet(new WalletConfig()
                 .withClientOptions(new ClientConfig().withNodes(Env.NODE))
-                .withSecretManager(new StrongholdSecretManager(Env.STRONGHOLD_PASSWORD, null, Env.STRONGHOLD_VAULT_PATH))
+                .withSecretManager(
+                        new StrongholdSecretManager(Env.STRONGHOLD_PASSWORD, null, Env.STRONGHOLD_VAULT_PATH))
                 .withCoinType(CoinType.Shimmer)
-                .withStoragePath(Env.STORAGE_PATH)
-        );
+                .withStoragePath(Env.STORAGE_PATH));
 
-        // Get account and sync it with the registered node to ensure that its balances are up-to-date.
+        // Get account and sync it with the registered node to ensure that its balances
+        // are up-to-date.
```

Comment:
I don't know if there is such a thing as a java format file but maybe we should have one otherwise we may oscillate between different IDEs preference whenever we edit things.

Reply:
Create an issue?

Reply:
Sure